platforms/chibios/flash_stm32.h

```diff
@@ -35,6 +35,7 @@ typedef enum { FLASH_BUSY = 1, FLASH_ERROR_PG, FLASH_ERROR_WRP, FLASH_ERROR_OPT,
 FLASH_Status FLASH_WaitForLastOperation(uint32_t Timeout);
 FLASH_Status FLASH_ErasePage(uint32_t Page_Address);
 FLASH_Status FLASH_ProgramHalfWord(uint32_t Address, uint16_t Data);
+FLASH_Status FLASH_ProgramDoubleWord(uint32_t Address, uint64_t Data);
```

Comment:
Let's match the preprocessor ifdef's as per the source file.

Reply:
`STM32L4XX` is defined in "lib/chibios/os/hal/ports/STM32/STM32L4xx/hal_lld.h", not included in this file. Adding the proprocessor will result in implicit declaration error of `FLASH_ProgramDoubleWord`